tests/Resources/require.test.js

```diff
@@ -291,4 +285,12 @@ describe('require()', function () {
 		const value = require('./circular.entry');
 		should.exist(value);
 	});
+	
```

Comment:
<!--
  1 failure:  tests/Resources/r...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/require.test.js;
  Line: 288;
-->
- :no_entry_sign: tests/Resources/require.test.js line 288 – Trailing spaces not allowed. (no-trailing-spaces)